ui/app/AppLayouts/Profile/views/wallet/MainView.qml

```diff
@@ -15,8 +19,89 @@ Column {
 
     signal goToNetworksView()
 
+
+    StatusBaseText {
+        id: titleText
+        text: qsTr("Wallet")
+        font.weight: Font.Bold
+        font.pixelSize: 28
```

Comment:
Hardcoded font sizes should be avoided -> `font.pixelSize: Style.current.primaryTextFontSize`